rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -89,8 +88,11 @@ def create(
         model_storage: ModelStorage,
         resource: Resource,
         execution_context: ExecutionContext,
+        cache_dir: str = None,
```

Comment:
this needs to be removed and instead the `cache_dir` should be added to the config in `get_default_config`

Reply:
Ok, it's done!